src/modules/video_player/index.js

```diff
@@ -19,6 +19,30 @@ function stepPlaybackSpeed(faster) {
     currentPlayer.player.setPlaybackRate(rates[idx]);
 }
 
+const handleVisibilityChange = (function() {
+    let userMuted = $('.player-button--volume .mute-button').length;
+
+    const clickHandler = () => {
+        userMuted = $('.player-button--volume .mute-button').length;
+    };
+
+    $('.player-button--volume').on('click', clickHandler);
+
+    return () => {
+        const button = $('.player-button--volume');
+
+        button.off('click', clickHandler);
+
+        if (document.hidden) {
+            $('.player-button--volume .mute-button').trigger('click');
+        } else if (!document.hidden && !userMuted) {
+            $('.player-button--volume .unmute-button').trigger('click');
```

Comment:
should we be using the player api directly instead of triggering clicks?

Reply:
@night I've updated the PR, i kept the page visibility api. Things to note:

* the document that is shown triggers visibility before the one that is hidden (that's why there's a settimeout in there, better ideea is more than welcome)
* could've used onBlur/onFocus, but the drawback is that you cannot watch a stream with sound on and do something on another monitor at the same time since you'll trigger blur
* if user mutes the player it will take it into account when switching tabs, `onbeforeunload` handles a case where `visibilitychange` triggers on refresh and breaks it

Reply:
Done